config/__base.json

```diff
@@ -0,0 +1,3 @@
+{
+	"discover_blog_id": 53424024
```

Comment:
Maybe we should name this file `shared.json`? The best approach would be to make `production.json` kind of shared file, which other config files would have to override. However we have at the moment two production envs for web and desktop app, so I don't think this is an option for us.